src/python/pants/base/exception_sink.py

```diff
@@ -237,16 +256,24 @@ def _iso_timestamp_for_now(cls):
   # NB: This includes a trailing newline, but no leading newline.
   _EXCEPTION_LOG_FORMAT = """\
 timestamp: {timestamp}
-args: {args}
+process title: {process_title}
+sys.argv: {args}
+bootstrap options: {bootstrap_options}
 pid: {pid}
 {message}
 """
 
   @classmethod
   def _format_exception_message(cls, msg, pid):
+    if cls._bootstrap_option_values:
```

Comment:
Rather than individually poking at fields of the bootstrap options, it would be good to capture the fields you need in set_bootstrap_options, and to have defaults for those fields.

Reply:
Truly fantastic idea. Will do.

Reply:
Done!